contracts/eosio.system/include/eosio.system/native.hpp

```diff
@@ -18,6 +18,16 @@ namespace eosiosystem {
    using eosio::public_key;
    using eosio::ignore;
 
+   /**
+    * @addtogroup eosiosystem
+    * @{
+    */
+   /**
+    * A weighted permission. 
+    * 
+    * @details Defines a weighted permission, that is a permission which has a weight associated. 
+    * A permission is defined by an account name plus a permission name.
+    */
    struct permission_level_weight {
```

Comment:
This is outside the scope of the documentation PR, but why are these things being defined twice in different files?

Reply:
it will be re-worked at some point as far as I know; for now they are public structs and we annotate them.